### PR TITLE
Add travis-ci config

### PR DESCRIPTION
**Pull request by jharrington22**
Add Travis CI Config.
- Targeting Node latest 4.2

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+language: node_js
+node_js:
+    - "4.0.0"
```
Comment (runk) [inline .travis.yml]: 4.2?

**Comment by camjc** [comment]
Good value 3 lines. Merging! :gem:

File: .travis.yml (hunk 1 above)
Comment (runk) [inline .travis.yml]: also set indentation to 2 spaces I guess

**Comment by camjc** [comment]
Merging!

**Pull request merged**